Detectors/TPC/qc/src/Tracks.cxx

```diff
@@ -207,6 +212,46 @@ bool Tracks::processTrack(const o2::tpc::TrackTPC& track)
+        mMapHist["hPhiAside"]->SetTitle("hPhiAside o2::base::Propagator not properly initialized");
+        mMapHist["hPhiCside"]->SetTitle("hPhiCside o2::base::Propagator not properly initialized");
+        mMapHist["hPhiBothSides"]->SetTitle("hPhiBothSides o2::base::Propagator not properly initialized");
+        mMapHist["hPt"]->SetTitle("hPt o2::base::Propagator not properly initialized");
+        mMapHist["hSign"]->SetTitle("hSign o2::base::Propagator not properly initialized");
+        mMapHist["hQOverPt"]->SetTitle("hQOverPt o2::base::Propagator not properly initialized");
+        mMapHist["hEtaNeg"]->SetTitle("hEtaNeg o2::base::Propagator not properly initialized");
+        mMapHist["hPhiAsideNeg"]->SetTitle("hPhiAsideNeg o2::base::Propagator not properly initialized");
+        mMapHist["hPhiCsideNeg"]->SetTitle("hPhiCsideNeg o2::base::Propagator not properly initialized");
+        mMapHist["hEtaPos"]->SetTitle("hEtaPos o2::base::Propagator not properly initialized");
+        mMapHist["hPtPos"]->SetTitle("hPtPos o2::base::Propagator not properly initialized");
+        mMapHist["hPhiAsidePos"]->SetTitle("hPhiAsidePos o2::base::Propagator not properly initialized");
+        mMapHist["hPhiCsidePos"]->SetTitle("hPhiCsidePos o2::base::Propagator not properly initialized");
+        mMapHist["h2DNClustersEta"]->SetTitle("h2DNClustersEta o2::base::Propagator not properly initialized");
+        mMapHist["h2DNClustersPhiAside"]->SetTitle("h2DNClustersPhiAside o2::base::Propagator not properly initialized");
+        mMapHist["h2DQOverPtPhiAside"]->SetTitle("h2DQOverPtPhiAside o2::base::Propagator not properly initialized");
+        mMapHist["h2DNClustersPhiCside"]->SetTitle("h2DNClustersPhiCside o2::base::Propagator not properly initialized");
+        mMapHist["h2DQOverPtPhiCside"]->SetTitle("h2DQOverPtPhiCside o2::base::Propagator not properly initialized");
+        mMapHist["h2DNClustersPt"]->SetTitle("h2DNClustersPt o2::base::Propagator not properly initialized");
+        mMapHist["h2DEtaPhi"]->SetTitle("h2DEtaPhi o2::base::Propagator not properly initialized");
+        mMapHist["h2DEtaPhiNeg"]->SetTitle("h2DEtaPhiNeg o2::base::Propagator not properly initialized");
+        mMapHist["hEtaVsPtNeg"]->SetTitle("hEtaVsPtNeg o2::base::Propagator not properly initialized");
+        mMapHist["hPhiVsPtNeg"]->SetTitle("hPhiVsPtNeg o2::base::Propagator not properly initialized");
+        mMapHist["h2DEtaPhiPos"]->SetTitle("h2DEtaPhiPos o2::base::Propagator not properly initialized");
+        mMapHist["hEtaVsPtPos"]->SetTitle("hEtaVsPtPos o2::base::Propagator not properly initialized");
+        mMapHist["hPhiVsPtPos"]->SetTitle("hPhiVsPtPos o2::base::Propagator not properly initialized");
+      }
+      return true;
+    }
+
+    if (dcaValue > mCutMaxAbsDCAr) {
+      return true;
+    }
```

Comment:
Shouldn't this be
```suggestion
    if (mUseCutMaxAbsDCArOnHistos && (dcaValue > mCutMaxAbsDCAr)) {
      return true;
    }
```

Reply:
actually, the dcaValue is signed, so one should check `std::abs(dcaValue)`

Reply:
That is totally true, I will push an update and fix this. Thanks a lot!